### PR TITLE
Update NoKillPlugin

### PR DESCRIPTION
**Pull request by Bluefissure**
It's not actively useful rn but in case the lobby blows up again

### CODE REVIEW

File: testing/live/NoKillPlugin/manifest.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[plugin]
+repository = "https://github.com/Bluefissure/NoKillPlugin.git"
+commit = "a361208fa5d732b5bcd31e34338c998fd1125a56"
+owners = ["Bluefissure"]
+project_path = "NoKillPlugin"
```
Comment (karashiiro) [inline testing/live/NoKillPlugin/manifest.toml:5]: You can remove this line, this refers to where your solution is stored.

Comment (NotNite) [reply testing/live/NoKillPlugin/manifest.toml:5]: I don't think you can? I'm pretty sure it's where the .csproj is stored, not the solution.

Comment (karashiiro) [reply testing/live/NoKillPlugin/manifest.toml]: Oh? That's how it seemed to work when I added PingPlugin and TTT. TTT has the solution file in `src`, so I had to adjust this field. PingPlugin didn't need the field at all.

**Comment by karashiiro** [comment]
https://github.com/Bluefissure/NoKillPlugin/blob/master/NoKillPlugin/NoKillPlugin.csproj#L36-L38

This time it's this messing up the workflow. DalamudPackager should do this already anyways.

**Comment by Bluefissure** [comment]
> https://github.com/Bluefissure/NoKillPlugin/blob/master/NoKillPlugin/NoKillPlugin.csproj#L36-L38
> 
> This time it's this messing up the workflow. DalamudPackager should do this already anyways.

Yeah, some of my plugins have weird legacy build flows that mess up the CI.

**Review by karashiiro — APPROVED** [review]

**Pull request merged**